include/chemfiles/Connectivity.hpp

```diff
@@ -279,11 +279,29 @@ inline bool operator>=(const Improper& lhs, const Improper& rhs) {
 /// other data are cached from it.
 class Connectivity final {
 public:
+
+    /// Stores the type of bond
+    enum BondOrder {
```

Comment:
Why not make this an enum class, outside of Connectivity ? The Connectivity class is more or less an implementation detail for users of chemfiles, and defining this as an enum inside Connectivity make it visible outside of the code.

Reply:
Hmm, where would be a good place for that? I thought about that but couldn't find a logical place for it.

Reply:
It could still be defined in the same file, together with `Bond`.

Reply:
Like a member of `Bond` or defined outside the class?

Reply:
Either way could work, and I don't have strong preference on this. Putting it inside Bond would be better with a plain enum, so that we can use `Bond::DOUBLE`, putting it outside would need an class enum, used as `BondOrder::DOUBLE`.

Reply:
I like it in `Bond` as a bond order is a property of a bond, not an independent concept.